src/styles/global-components.js

```diff
@@ -26,5 +27,11 @@ module.exports = {
         @media (min-width: 600px) {
             padding: 0 24px;
         }
+    `,
+
+    UserListItem: Styled(ListItem)`
+        width: 100%;
+        max-width: 360px;
+        background-color: ${Theme.palette.background.paper} !important;
```

Comment:
I'm interested to know if there's a way for me to omit the `!important` flag here. This style works as expected normally, but when the `ListItem` has the `button` prop set, the `background-color` is changed somewhere else with a higher specificity.

I do like the idea of using the `theme` color to make it easier to support dark themes and whatnot later. It seems like this may become increasingly relevant on the web in years to come now that the latest macOS update has started a conversation around future detection & implementation.